Src/BootCamp.Chapter/Program.cs

```diff
@@ -1,12 +1,49 @@
-﻿using System;
+using System;
 
 namespace BootCamp.Chapter
 {
     class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            GetDetails();
```

Comment:
Function name could be better. You're actually calculating or prompting for Bmi.